spec/suites/layer/vector/PolygonSpec.js

```diff
@@ -156,13 +156,22 @@ describe('Polygon', function () {
 			expect(layer.getCenter()).to.be.nearLatLng(L.latLng([5, 5]), 1e-1);
 		});
 
+		it('should compute center of a big simple polygon around equator with maxZoom', function () {
+			var latlngs = [
+				[[0, 0], [10, 0], [10, 10], [0, 10]]
+			];
+			map.setMaxZoom(18);
+			var layer = new L.Polygon(latlngs).addTo(map);
+			expect(layer.getCenter()).to.be.nearLatLng(L.latLng([5, 5]), 1e-1);
+		});
+
 		it('should compute center of a small simple polygon', function () {
 			var latlngs = [
 				[[0, 0], [0.010, 0], [0.010, 0.010], [0, 0.010]]
 			];
 			var layer = new L.Polygon(latlngs).addTo(map);
 			map.setZoom(0);  // Make the polygon disappear in screen.
```

Comment:
You should then restore the zoom, here or in `afterEach` hook.

Reply:
I changed it to `beaforeEach` so for each test the map is new created and the zoom & center will be set